makejdk.sh

```diff
@@ -186,13 +184,14 @@ cloneOpenJDKGitRepo()
 testOpenJDKViaDocker()
 {
   if [[ ! -z $JTREG ]]; then
-    docker run --privileged -t -v "${WORKING_DIR}/${OPENJDK_REPO_NAME}":/openjdk/jdk8u/openjdk --entrypoint jtreg.sh "${CONTAINER}"
+    mkdir -p "${WORKING_DIR}/target"
+    docker run -t -v "${WORKING_DIR}/${OPENJDK_REPO_NAME}":/openjdk/jdk8u/openjdk -v "${WORKING_DIR}/sbin":/openjdk/sbin -v "${WORKING_DIR}/target":/openjdk/target --entrypoint /openjdk/sbin/jtreg.sh "${CONTAINER}"
```

Comment:
In `/openjdk/jdk8u/openjdk`, not clear why we have subfolder `openjdk` inside `/openjdk/jdk8u/` maybe its a good time to resolve this